### DISCUSSION TITLE
4.4.1 render style addon

### DISCUSSION THREAD

**Discussion by cricri-du-lauragais** [osmandapp/OsmAnd#16699] [Maps Related]
hello, since version 4.4.1 (and maybe version 4.4.0) the addon styles (xxxx.addon.render.xml) overwrite the definition of the elements in the personal styles (custom render).
if you disable or delete the xxx.addon.render.xml, the standard styles do not display the elements anymore (normal)
and at each update the xxx.addon.render.xml are reloaded.
how to manage these addon.render.xml which we can't change the name, can we create specific ones linked to a particular custom style ?
or choose if a personal style can avoid them ?

**Comment by cricri-du-lauragais** [comment]
this only happens with OpenGL

**Reply by dmpr0** [reply]
Hi! Could you please provide custom rendering style for test?

**Comment by cricri-du-lauragais** [comment]
Hello, 
https://github.com/OsmAnd-Rendering/Motorcycle/blob/main/enduro.render.xml
texte size of contour lines
text color/size of mtb route
order of "routes" for mtb/hiking
etc ;)

**Comment by XandrexOSM** [comment]
hi @dmpr0 , have you had the chance to reproduce the issue ?

This behaviour was "okay" when it occured on beta version.
Now that 4.4 is the official version on the GooglePlay store and on the Apple AppStore, it could really prevent people from applying the upgrade.

If there is a workaround or a way for the third-party render devs to tweak on their side, it would ease things a bit.

**Comment by xmd5a2** [comment]
Confirmed.
Legacy engine:
![Screenshot_2023-04-06_12-32-29](https://user-images.githubusercontent.com/5829014/230337339-d83c3103-cf88-41fd-81ac-bb2f9a31a5d1.png)

OpenGL engine:
![Screenshot_2023-04-06_12-32-41](https://user-images.githubusercontent.com/5829014/230337401-244eff26-4b8c-4f27-ad2a-b385b225c111.png)

https://www.openstreetmap.org/#map=16/55.6290/37.5748
@dmpr0

**Comment by cricri-du-lauragais** [comment]
now everything works correctly with version 4.4.7.
thanks a lot ;)
i guess the order of interpretation of render files is now addon first and then style?
